nullaway/src/test/resources/com/uber/nullaway/testdata/NullAwayNegativeCases.java

```diff
@@ -809,4 +809,12 @@ static void testTwoParamIter() {
       s.hashCode();
     }
   }
+
+  static int testNoCrashOnShifts(int n) {
```

Comment:
Does this really test the relevant code path?  I think you need to have some kind of unboxing going on.  E.g., make the method return `Integer` and return an unsigned right shift expression

Reply:
The original example is just a shift inside a switch, for an int instance field. But you are right, this test doesn't actually trigger that path. Fixing it now, the new version does fail when I remove the added line in NullAway.java :)